playbook-website/app/controllers/pages_controller.rb

```diff
@@ -6,6 +6,15 @@
 
 require_relative "application_controller"
 
+# Need Wade's input
+require "will_paginate"
+require "playbook/pagination_renderer"
+# Since we don't require it for clients should we make steps for them...
+# To require it if they want it? Boostrap renderer works this way
+# Nitro Example: https://github.com/powerhome/nitro-web/blob/master/components/nitro_theme/lib/nitro_theme.rb#L8-L10
```

Comment:
My input on what exactly?

Reply:
Is this ok to leave to the client to include If they want it? I assume yes but wanted to check @wadewinningham